torch/optim/optimizer.py

```diff
@@ -1,11 +1,15 @@
 from collections import defaultdict
-from torch._six import container_abcs
-
-import torch
+from typing import Iterable, Union
 from copy import deepcopy
 from itertools import chain
 import warnings
 
+import torch
+from .. import Tensor
```

Comment:
minor: you may just use `Tensor = torch.Tensor` or use `torch.Tensor` on line 12 instead, a little faster.